app/assets/javascripts/dashboard/dashboard.js

```diff
@@ -1062,3 +1063,11 @@ function requestNodeForceRemoval(minionId) {
     .fail(enableOrchTriggers)
     .fail(notifyRemovalError);
 }
+
+function onlineHtml(minion) {
+  if(minion.online) {
+    return '<i class="fa fa-circle text-success fa-2x" aria-hidden="true"></i>';
```

Comment:
Title attribute with `Online` would be nice.

Reply:
good idea! added